lib/tasks/import.rake

```diff
@@ -0,0 +1,57 @@
+namespace :reports do
+  task :import => :environment do
+    CSV.foreach("logs.csv", headers: true) do |l|
+      log = Log.new
+      log.id = l['id']
+      log.source_id = l['source_id']
+      log.message_id = l['message_id']
+      log.report_id = l['report_id']
+      log.level_id = l['level_id']
+      User.without_auditing do
+        User.as_anonymous_admin do
+          log.save!
+        end
+      end
+    end
+
+    CSV.foreach("sources.csv", headers: true) do |s|
+      source = Source.new
+      source.id = s['id']
+      source.value = s['value']
+      User.without_auditing do
+        User.as_anonymous_admin do
+          source.save!
+        end
+      end
+    end
+
+    CSV.foreach("messages.csv", headers: true) do |m|
+      message = Message.new
+      message.id = m['id']
+      message.value = m['value']
+      User.without_auditing do
+        User.as_anonymous_admin do
+          message.save!
+        end
+      end
+    end
+
+    CSV.foreach("reports.csv", headers: true) do |r|
+      report = ConfigReport.new
+      report.id = r['id']
+      report.host_id = r['host_id']
+      report.reported_at = r['reported_at']
+      report.created_at = r['created_at']
+      report.updated_at = r['updated_at']
+      report.status = JSON.parse(r['status'])
+      report.metrics = JSON.parse(r['metrics'])
+      report.type = r['type']
+      report.origin = r['origin']
+      User.without_auditing do
+        User.as_anonymous_admin do
+          report.save!
+        end
+      end
+    end
```

Comment:
At the end of each file, plese set PG sequence to the highest seen ID + 1: https://www.postgresql.org/docs/9.1/sql-altersequence.html